test/builtin_dataset_mocks.py

```diff
@@ -504,3 +504,41 @@ def imagenet(info, root, config):
     make_tar(root, f"{devkit_root}.tar.gz", devkit_root, compression="gz")
 
     return num_samples
+
+
+@dataset_mocks.register_mock_data_fn
+def sintel(info, root, config):
```

Comment:
@pmeier - I wasn't very sure what you meant by "archive generation" in your previous comment, but I tried replicating the test. Could you please take a look at this function and suggest if this does the job?

I ran the test using `python3 test/test_prototype_builtin_datasets.py -k sintel` and the test seems to pass (no failures). But I just want to double check.

Reply:
The test has to be run by `pytest` otherwise no test is run. If you do 

```bash
pytest test/test_prototype_builtin_datasets.py -k sintel
```

you will see [these failures](https://app.circleci.com/pipelines/github/pytorch/vision/12637/workflows/2819215a-ad45-4248-b753-67feed8123f5/jobs/1001187) popping up.